app/server.go

```diff
@@ -22,16 +23,19 @@ type KubeRouter struct {
 
 func NewKubeRouterDefault(config *options.KubeRouterConfig) (*KubeRouter, error) {
 
-	if len(config.Master) == 0 && len(config.Kubeconfig) == 0 {
-		if _, err := os.Stat("/var/lib/kube-router/kubeconfig"); os.IsNotExist(err) {
-			return nil, errors.New("Either one of --master or --kubeconfig must be specified. Or valid kubeconfig file must exist as /var/lib/kube-router/kubeconfig")
+	var clientconfig *rest.Config
+	var err error
+	// Use out of cluster config if the URL or kubeconfig have been specified. Otherwise use incluster config.
+	if len(config.Master) != 0 && len(config.Kubeconfig) != 0 {
```

Comment:
Ignore the travis failure. attempt to push docker image is failing for the PR's submitted from external repos. Please see #84